JumpscaleCore/clients/test/test_sonic_client.py

```diff
@@ -0,0 +1,318 @@
+import unittest
+from Jumpscale import j
+from random import randint
+from base_test import BaseTest
+
+
+@unittest.skip("https://github.com/threefoldtech/jumpscaleX_core/issues/250")
+class SonicClient(BaseTest):
+
+    @classmethod
+    def setUpClass(cls):
+        cls.info("install sonic builder")
+        j.builders.apps.sonic.install()
+
+        cls.info("Start Sonic server")
+        j.servers.sonic.default.start()
+
+        cls.info("create a Sonic client")
+        cls.client = j.clients.sonic.get('test', host="127.0.0.1", port=1491, password='123456')
+
+    def setUp(self):
+        print('\t')
+        self.info('Test case : {}'.format(self._testMethodName))
+
+        self.data = {
+            'post:1': "this is some test text hello",
+            'post:2': 'this is a hello world post',
+            'post:3': "hello how is it going?",
+            'post:4': "for the love of god?",
+            'post:5': "for the love of a lord?",
+        }
+
+        self.COLLECTION = "COLL_{}".format(randint(1, 1000))
+        self.BUCKET = "BUCKET_{}".format(randint(1, 1000))
+
+    def tearDown(self):
+        self.info("Flush all data in {} collection, and {} bucket".format(self.COLLECTION, self.BUCKET))
+        self.client.flush(self.COLLECTION, self.BUCKET)
+
+    @classmethod
```

Comment:
Please remove this tearDown .. not good specially if other services are using sonic